config/locales/en.yml

```diff
@@ -147,6 +147,8 @@ en:
       city: City
       state: State
       zipcode: ZIP Code
+    errors:
+      duplicate_ssn: 'You may already have an account, please use that one. You can always change your email address in your existing account.'
```

Comment:
Perhaps more explicit?  `An account has already been created using this SSN.`?  I guess the wording can be tweaked later.